tools/ci_build/github/linux/docker/Dockerfile.package_ubi8_cuda11_8_tensorrt8_6

```diff
@@ -0,0 +1,45 @@
+# --------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------
+# Dockerfile to Test ONNX Runtime on UBI8 with CUDA 11.8 and TensorRT 8.6
+
+# Build base image with required system packages
+FROM nvidia/cuda:11.8.0-cudnn8-devel-ubi8 AS base
+
+ENV PATH /usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/src/tensorrt/bin:${PATH}
+
+RUN dnf update -y &&\
```

Comment:
As I said in another PR, you do not need to call "dnf update". If here it is apt-get, you should. "apt-get update" is to fetch package metadata. But "dnf upgdate" upgrades packages. It means when you run this command, the cuda packages could be upgraded to a newer version (you haven't call "dnf versionlock" yet).

Reply:
thx, updated!